_shows/17_18/the_effect.md

```diff
@@ -26,7 +26,7 @@ crew:
   name: Rosie Hudson
 - role: Technical Director
   name: Ben Woodford
-- role: Shadow Techincial Director
+- role: Shadow Techincal Director
```

Comment:
You made it slightly better, but it should be Technical

Reply:
LOL